requirements_dev.txt

```diff
@@ -13,6 +13,7 @@ pytest-vcr==1.0.2
 vcrpy==4.3.1
 freezegun==1.2.2
 flaky==3.7.0
+setuptools-scm==7.1.0  # needed by package.py
```

Comment:
hey hello, thanks `requirements_dev` is only for development. For packaging we also need these 3 packages:

```
packaging wheel setuptools-scm
```

Which we install during the packaging build job (if you check the github actions) only for that vm, or locally if we build locally.

What may make sense in that case is another requirements_packaging.txt to specify those in one common place?

What do you think @kelsos ?

Reply:
Hi, thanks for your comment.
My bad, I missed the workflows ...:(

Reply:
> hey hello, thanks `requirements_dev` is only for development. For packaging we also need these 3 packages:
> 
> ```
> packaging wheel setuptools-scm
> ```
> 
> Which we install during the packaging build job (if you check the github actions) only for that vm, or locally if we build locally.
> 
> What may make sense in that case is another requirements_packaging.txt to specify those in one common place?
> 
> What do you think @kelsos ?

Yes having a requirements_packaging.txt would make sense.